uint32 action_symbol
uint32 decision_episode
